### PR TITLE
Test python_workers compat flag in imports test

### PR DESCRIPTION
**Pull request by danlapid**

### CODE REVIEW

File: build/ci.bazelrc
Comment (hoodmane) [inline build/ci.bazelrc:1]: What does this change do?

**Review by hoodmane — APPROVED** [review]
Looks good thank!

File: build/ci.bazelrc
Comment (danlapid) [reply build/ci.bazelrc:1]: Makes python tests run in all ci-test jobs instead of only linux test jobs

Comment (danlapid) [reply build/ci.bazelrc:1]: Trying to repro the failures seen in. 
https://github.com/cloudflare/workers-sdk/actions/runs/12132349789/job/33826752898?pr=7418
and
https://github.com/cloudflare/workers-sdk/actions/runs/12136017044/job/33837024988?pr=7418

**Pull request merged**